charts/incubator/mineos/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
-  repository: tccr.io/truecharts/mineos
+  repository: hexparrot/mineos
   pullPolicy: IfNotPresent
-  tag: latest@sha256:dae5c70dd0ec272707cf2eca7758f635ef4d76a177b437e2f7651b6bbcbc0245
+  tag: "latest"
```

Comment:
WHY the heck is that reverted?

Reply:
```
Run tools/build-release.sh
No owner configured, defaulting to truecharts
env #CR_TOKEN not found, defaulting to production=false
No repo configured, defaulting to apps
Looking up latest tag...
Discovering changed charts since 'babybuddy-8.0.16'...
Error: looks like "https://truecharts.org" is not a valid chart repository or cannot be reached: failed to fetch https://truecharts.org/index.yaml : [4](https://github.com/truecharts/apps/runs/7280466302?check_suite_focus=true#step:6:5)04 Not Found
Error: Process completed with exit code 1.
```

Reply:
Please start reading the logs and stages.

This only catalog tests.

App installation test passed without problems.

Reply:
ok, i will re add it back....but

```
 repository: hexparrot/mineos
```

or 

```
  repository: tccr.io/truecharts/mineos
```